### PR TITLE
PowerSubscriber outputs messages according to received message

### PR DESCRIPTION
**Pull request by ramesses2nd**
I add a feature for PowerSubscriber to output messages according to received messages from publisher.

It means:
- power-on when receiving `1`
- power-off when receiving `0`
- error otherwise

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 build/
 install/
 log/
-
+.vscode/
```
Comment (poifull10) [inline .gitignore]: Used editors depend on users. so `.vscode/` should be written to user's own ignore lists.
Please remove this statement and add it to your `~/.config/git/ignore` .
(And please remember add null statement in all files 👍 )

File: src/power_subscriber.hpp [hunk 2]
```diff
@@ -15,13 +15,24 @@ class PowerSubscriber : public rclcpp::Node
   PowerSubscriber() : rclcpp::Node("power_subscriber")
   {
     subscription_ = this->create_subscription<std_msgs::msg::String>(
-      "power_onoff", 10, std::bind(&PowerSubscriber::topic_callback, this, _1));
+      "power_ops", 10, std::bind(&PowerSubscriber::topic_callback, this, _1));
```
Comment (poifull10) [inline src/power_subscriber.hpp]:
```suggestion
      "power_ops", 30, std::bind(&PowerSubscriber::topic_callback, this, _1));
```

**Comment by poifull10** [comment]
please `git rebase -i origin/master` to get in order dirty commits.

File: .gitignore (hunk 1 above)
Comment (ramesses2nd) [reply .gitignore]: I resolved this one

File: .gitignore [hunk 3]
```diff
@@ -1,4 +1,3 @@
 build/
 install/
-log/
-
+log/
```
Comment (poifull10) [inline .gitignore]: please add null line :)
```suggestion
log/

```

File: src/power_subscriber.hpp [hunk 4]
```diff
@@ -15,13 +15,24 @@ class PowerSubscriber : public rclcpp::Node
...
+    {
+      RCLCPP_INFO(this->get_logger(), "Receive power-on");
+    }
+    else if (msg->data == "0")
+    {
+      RCLCPP_INFO(this->get_logger(), "Receive power-off");
+    }
+    else
+    {
+      RCLCPP_ERROR(this->get_logger(), "Receive unknown string '%s'", msg->data.c_str());
```
Comment (poifull10) [inline src/power_subscriber.hpp]:
```suggestion
      RCLCPP_ERROR(this->get_logger(), "Receive unknown string " + msg->data);
```

**Comment by poifull10** [comment]
Thanks for your contribution.

**Pull request merged**